package.json

```diff
@@ -15,38 +15,53 @@
         "dist"
     ],
     "scripts": {
-        "build": "webpack --mode production",
-        "watch": "webpack --watch --mode development",
+        "build": "npm-run-all -s \"create-version\" \"build-wp-dev\"",
+        "build-prod": "npm-run-all -s \"clean-dist\" \"create-version\" \"build-wp-prod\"",
+        "build-sdk": "npm-run-all -s \"clean-dist\" \"create-version\" \"build-wp-prod\"",
+        "build-wp-prod": "webpack --mode production",
+        "build-wp-dev": "webpack --mode development",
```

Comment:
I'm not convinced about adding in these two commands. I think it would be cleaner to in-line them in `build` and `build-prod` so as not to clutter up the package.json.

Reply:
npm-run-all does several "npm run"s. The organization would have to be different if one tried to 'inline' "build-wp-dev". That is, the command can't be inlined using npm-run-all.

I don't see how breaking out the webpack operation is all that confusing to anyone.